build/tutorials/exchanges/exchange-c-chain-integration.md

```diff
@@ -0,0 +1,70 @@
+# Avalanche C-Chain Integration Guide
+
+## Overview
+
+The objective of this document is to provide a brief overview of how to integrate with the EVM-Compatible Avalanche C-Chain. For teams that already support ETH, supporting the C-Chain is as straightforward as spinning up an Avalanche node (which has the same API as go-ethereum) and populating Avalanche’s ChainID (43114) when constructing transactions.
```

Comment:
Can we add a link for "same API" to https://eth.wiki/json-rpc/API and https://geth.ethereum.org/docs/rpc/server